lib/features/profiles/screens/wallet_screen.dart

```diff
@@ -51,6 +52,14 @@ class _WalletScreenState extends State<WalletScreen>
     await context.read<ProfilesCubit>().fetchProfiles(parentGuid);
   }
 
+  Future<String> _getAppIDAndVersion() async {
+    final packageInfo = await PackageInfo.fromPlatform();
+    final result =
+        '${packageInfo.packageName} v${packageInfo.version}(${packageInfo.buildNumber})';
```

Comment:
Do we wanna use the same format as in the Givt app?

          '''${context.l10n.appVersion} ${packageInfo.version}.${packageInfo.buildNumber}''',

Reply:
I assume the first part ai a localised prefix with some text?
Also, there is no app id in the Givt app option, I think it can be useful for us as well